registry.k8s.io/images/k8s-staging-capi-ibmcloud/images.yaml

```diff
@@ -20,3 +20,8 @@
     "sha256:279e1df7efa64066396de02fbcc6a917ae23008cbcac7bce7a120ca09b1c9da1": ["v0.8.0"]
     "sha256:c7c571d112ccb804acc6fe81ec9eb4a741515f576439805b33269a418a139d6b": ["v0.9.0"]
     "sha256:112910167f407582e359cebe1c77c93f37deab69220a5cb62135ccf5e16d55c5": ["v0.10.0"]
+- name: powervs-cloud-controller-manager
+  dmap:
+    "sha256:5c311b5956141d6131397d5208edbfa8682a06ac428302bf80d759b122ed29ac": ["6256ccf"]
+    "sha256:1fe88a1aa84df88e5f234fd185a45bc41b3d79b6f7d7f31c6d944d265698f9cc": ["6256ccf_linux_ppc64le"]
+    "sha256:85a45c17cb16808cab25f72801639c546fe03a367492d411a222722b6e0afcf1": ["6256ccf_linux_amd64"]
```

Comment:
no need to include the individual arch image references, please remove these 2 entries.